hyperspy/signal.py

```diff
@@ -2835,6 +2835,18 @@ def get_1D_sum_explorer_wrapper(*args, **kwargs):
                 )
             return np.nan_to_num(to_numpy(navigator.data)).squeeze()
 
+        def get_dynamic_image_explorer(*args, **kwargs):
+            am = self.axes_manager
+            nav_ind = am.indices[2:]  # image at first 2 nav indices
+            slices = [slice(None)] * len(am.navigation_axes)
+            slices[2:] = nav_ind
+            if len(navigator.axes_manager.signal_axes) > 0:
+                new_nav = navigator.transpose(signal_axes=len(am.navigation_axes)) # roll axes to signal axes
+            else:
+                new_nav=navigator
+            ind = new_nav.isig.__getitem__(slices=slices)  # Get the value from the nav reverse because hyperspy
```

Comment:
comment incomplete?

Reply:
More of just a terrible comment :) It took me a while to figure out what I was even trying to say so I might just delete it.

I just struggled to figure out if the index should be reversed or not because we are using the hyperspy index an not the array index.